tests/mdev_configuration_allocation_test.go

```diff
@@ -144,7 +150,7 @@ var _ = Describe("[Serial][sig-compute]MediatedDevices", func() {
 			tests.UpdateKubeVirtConfigValueAndWait(config)
 
 			By("Verifying that an expected amount of devices has been created")
-			checkAllMDEVCreated(desiredMdevTypeName, expectedInstancesNum)
+			Eventually(checkAllMDEVCreated(desiredMdevTypeName, expectedInstancesNum), 3*time.Minute, 15*time.Second)
```

Comment:
Aren't you missing the matcher part ? i.e. the `Should(...)` in `Eventually(<xyz>).Should(<matcher>)`

The same applies for the same entries below.